package.json

```diff
@@ -55,11 +55,6 @@
       "docs/"
     ]
   },
-  "babel": {
```

Comment:
Actually, nevermind, I just checked why I added that from git blame and it was when I added the `dist` task. Can you run `npm run dist` without that? It was minification that exploded not rollup.

Did you nuke rollup and install it again? Maybe you have an old version?